### PR TITLE
Show compiletime Qt version in aboutsettings

### PR DESCRIPTION
**Pull request by agilob**
This won't conflict with new AV so can be merged safely.

twitchinstallarch1

### CODE REVIEW

**Comment by antis81** [comment]
:+1:

**Comment by agilob** [comment]
I'm thinking now, we could add a "Copy debug" button in advanced, when user clicks it, all necessary for us debug info is copied ready to paste on github.

**Comment by antis81** [comment]
Maybe reorganize table? Example:

| ~~Ignore header~~ | ~~required for Github~~ |
| --- | --- |
| **You are using qTox nightly build.** | toxcore version: 0.0.0 |
| Commit: 01234abcdef… | Qt version: 5.5.1 |

A "Copy this information" button would be nice, yes.

**Comment by agilob** [comment]
I'll think about it, I would prefer this to be max. effortless for users. Please merge this and #2494 if you dont see a reason why it shouldn't be merged ;)

File: src/widget/form/settings/aboutform.cpp [hunk 1]
```diff
@@ -46,6 +46,8 @@ void AboutForm::replaceVersions()
                                                        QString::number(TOX_VERSION_MAJOR) + "." +
                                                        QString::number(TOX_VERSION_MINOR) + "." +
                                                        QString::number(TOX_VERSION_PATCH)));
+    bodyUI->qtVersion->setText(
+                bodyUI->qtVersion->text().replace("$QTVERSION", QT_VERSION_STR));
```
Comment (antis81) [inline src/widget/form/settings/aboutform.cpp]: Why so complicated? Simple:

``` c++
body->qtVersion->setText(QString::fromUtf8("Qt version: %1").arg(QT_VERSION_STR));
```

File: src/widget/form/settings/aboutsettings.ui [hunk 2]
```diff
@@ -134,6 +134,13 @@
             </property>
            </widget>
           </item>
+          <item row="2" column="1">
+           <widget class="QLabel" name="qtVersion">
+            <property name="text">
+             <string>Qt version: $QTVERSION</string>
```
Comment (antis81) [inline src/widget/form/settings/aboutsettings.ui]: Maybe `Qt version: n.a.` as default.

File: src/widget/form/settings/aboutform.cpp (hunk 1 above)
Comment (agilob) [inline src/widget/form/settings/aboutform.cpp]: Tried to avoid %1 in designer mode, so it's more obvious when editing XML in UI.

twitchinstallarch3

Comment (antis81) [inline src/widget/form/settings/aboutform.cpp]: Just set string in code -> all fine.

**Comment by agilob** [comment]
Yup, that's simpler and faster.

**Pull request merged**